.github/workflows/promptflow-release-testing-matrix.yml

```diff
@@ -13,28 +13,58 @@ on:
         required: false
         type: string
 env:
-  packageSetupType: promptflow_with_extra
   testWorkingDirectory: src/promptflow
   PYTHONPATH: ${{ github.workspace }}/src/promptflow
   IS_IN_CI_PIPELINE: "true"
 jobs:
-  id:
+  build:
     runs-on: ubuntu-latest
     steps:
-      - name: workflow run id - ${{ inputs.id }}
-        run: |
-          echo "workflow run id: ${{ inputs.id }}"
-  all_tests:
+    - name: workflow run id - ${{ inputs.id }}
```

Comment:
how about keeping this step as a separate job?

this step is to identity current worflow run, so that our release pipeline can know, so it's not related to build.

Reply:
OK